execution/scan/vector_selector.go

```diff
@@ -49,6 +49,12 @@ type vectorSelector struct {
 	numShards int
 }
 
+type point struct {
```

Comment:
Do we still need this new struct?

Reply:
well I made this just to have a good relation between timestamp and values in our data.

Reply:
what changes do we need now ?